functions/Get-DbaPrivilege.ps1

```diff
@@ -0,0 +1,122 @@
+
+      Requires Local Admin rights on destination computer(s).
+
+      .PARAMETER ComputerName
+      The SQL Server (or server in general) that you're connecting to. This command handles named instances.
+
+      .PARAMETER Credential
+      Credential object used to connect to the computer as a different user.
+
+      .NOTES
+      Author: Klaas Vandenberghe ( @PowerDBAKlaas )
+      Tags: Privilege
+      dbatools PowerShell module (https://dbatools.io)
+      Copyright (C) 2016 Chrissy LeMaire
+      This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+      This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+      You should have received a copy of the GNU General Public License along with this program. If not, see http://www.gnu.org/licenses/.
+
+      .LINK
+      https://dbatools.io/Get-DbaPrivilege
+
+      .EXAMPLE
+      Get-DbaPrivilege -ComputerName sqlserver2014a
+
+      Gets the local privileges on computer sqlserver2014a.
+
+      .EXAMPLE   
+      'sql1','sql2','sql3' | Get-DbaPrivilege
+
+      Gets the local privileges on computers sql1, sql2 and sql3.
+
+      .EXAMPLE
+      Get-DbaPrivilege -ComputerName sql1,sql2 | Out-Gridview
+
+      Gets the local privileges on computers sql1 and sql2, and shows them in a grid view.
+
+  #>
+  [CmdletBinding()]
+  Param (
+    [parameter(ValueFromPipeline)]
+    [Alias("cn","host","Server")]
+    [string[]]$ComputerName = $env:COMPUTERNAME,
+    [PSCredential] [System.Management.Automation.CredentialAttribute()]$Credential
+  )
+
+BEGIN
+  {
+    function Convert-SIDToUserName ([string] $SID ) {
+      $objSID = New-Object System.Security.Principal.SecurityIdentifier ("$SID") 
+      $objUser = $objSID.Translate( [System.Security.Principal.NTAccount]) 
+      $objUser.Value
+    }
+    $FunctionName = (Get-PSCallstack)[0].Command
+    $ComputerName = $ComputerName | ForEach-Object {$_.split("\")[0]} | Select-Object -Unique
+  }
+PROCESS
+  {
+    foreach ($computer in $ComputerName)
+    {
+      Write-Verbose "$FunctionName - Connecting to $computer"
+      if ( Test-PSRemoting -ComputerName $Computer )
+      {
+        Write-Verbose "$FunctionName - Getting Privileges on $Computer"
+        $Priv = $null
+        $Priv = Invoke-Command -ComputerName $computer -ScriptBlock {secedit /export /cfg C:\Windows\Temp\secpol.cfg > $NULL ;
+        Get-Content C:\Windows\Temp\secpol.cfg | Where-Object { $_ -match "SeBatchLogonRight" -or $_ -match 'SeManageVolumePrivilege' -or $_ -match 'SeLockMemoryPrivilege' }}
```

Comment:
yeah, i would do $env:temp or $temp = ([System.IO.Path]::GetTempPath()).TrimEnd("\")  here. $env:temp is shorter but it uses DOS formatting.